hazelcast-command-line/src/main/java/com/hazelcast/commandline/HazelcastCommandLine.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.commandline;
+
+import picocli.CommandLine;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.nio.charset.StandardCharsets;
+import java.time.Clock;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static com.hazelcast.instance.BuildInfoProvider.getBuildInfo;
+import static com.hazelcast.internal.util.StringUtil.isNullOrEmpty;
+import static picocli.CommandLine.Command;
+import static picocli.CommandLine.Option;
+
+/**
+ * Main command class for Hazelcast operations
+ */
+@Command(name = "hz", description = "Utility for the Hazelcast operations." + "%n%n"
+        + "Global options are:%n", versionProvider = VersionProvider.class, mixinStandardHelpOptions = true, sortOptions = false)
+class HazelcastCommandLine
```

Comment:
Rename to HazelcastServerCommandLine

Reply:
Done: https://github.com/hazelcast/hazelcast/pull/20262/commits/3a2febad877f0d6184491f527081246c26105cc8